multinode-demo/validator.sh

```diff
@@ -182,6 +182,9 @@ while [[ -n $1 ]]; do
     elif [[ $1 == --block-production-method ]]; then
       args+=("$1" "$2")
       shift 2
+    elif [[ $1 == --wen-restart ]]; then
```

Comment:
we need this in `multinode-demo/bootstrap-validator.sh` as well right?

Reply:
Done.